run_unittests.py

```diff
@@ -135,7 +135,7 @@ def main():
         # Let there be colors!
         if 'CI' in os.environ:
             pytest_args += ['--color=yes']
-        pytest_args += ['./run_unittests.py']
+        pytest_args.append(str(_pathlib.Path(__file__).resolve(strict=True)))
```

Comment:
Why jump through hoops to convert `__file__` to several different types of objects and then right back to itself?
```suggestion
        pytest_args += [__file__]
```